### PR TITLE
Fixed windows paths

### PR DESCRIPTION
**Pull request by DNedic**
Fix windows paths that break building on windows, most notably because of static asserts. Backslashes were not properly escaped. In the abscence of a better fix, this replaces backslashes with double backslash in jinja templates, as backslashes should not be used on linux anyway.

### CODE REVIEW

**Comment by thirtytwobits** [comment]
Please rebase. I've fixed our CI and upstream dependency issues. Thanks.

File: README.rst [hunk 1]
```diff
@@ -102,7 +102,7 @@ Partial example: generating a C struct
         *
         * Auto-generated, do not edit.
         *
-        * Source file: {{T.source_file_path}}
+        * Source file: {{T.source_file_path | replace("\\", "\\\\")}}
```
Comment (thirtytwobits) [inline README.rst:105]: We wouldn't need to do this if source_file_path was a `pathlib.Path`, right? Perhaps that's the better fix; to convert this into pathlib?

Comment (DNedic) [reply README.rst:105]: This is a quick fix, the issue has been documented for over a year. There is plenty of time to fix it properly afterwards, i may even give it a shot.

Comment (pavel-kirienko) [reply README.rst:105]: >We wouldn't need to do this if source_file_path was a pathlib.Path, right? Perhaps that's the better fix; to convert this into pathlib?

Check this out: https://github.com/UAVCAN/pydsdl/pull/77

We would have to use `.as_posix()` though to ensure consistent behavior.

**Comment by thirtytwobits** [comment]
Cool. We'll take this and then do #243 later.

**Pull request merged**